DataFormats/Detectors/TRD/src/LinkRecord.cxx

```diff
@@ -24,6 +24,7 @@ uint32_t LinkRecord::getHalfChamberLinkId(uint32_t detector, uint32_t rob)
   int stack = (detector % constants::NLAYER);
   int layer = ((detector % (constants::NLAYER * constants::NSTACK)) / constants::NLAYER);
   int side = rob % 2;
+  std::cout << "sector, stack, layer, side :" << sector <<" " << stack << " " << layer << " " << side << std::endl;
```

Comment:
Easy one: remove the cout

Reply:
I suspect there are a few LOG(info)'s around as well